starlark/value.go

```diff
@@ -927,6 +928,49 @@ func (s *Set) Union(iter Iterator) (Value, error) {
 	return set, nil
 }
 
+// A Module is a named collection of values,
+// typically a suite of functions imported by a load statement.
+//
+// It differs from go.starlark.net/starlarkstruct.Struct primarily
+// in that its string representation does not enumerate its fields.
```

Comment:
This doesn't seem like sufficient motivation to introduce a new type, especially a builtin type (`starlarkstruct` is not part of the spec or the main package). But I think I'm missing a lot of stargo context.